src/modules/Dropdown/Dropdown.js

```diff
@@ -420,7 +420,11 @@ export default class Dropdown extends Component {
       if (!this.isMouseDown) {
         const { closeOnBlur } = this.props
         debug('mouse is not down, closing')
-        if (closeOnBlur) this.close()
+        if (closeOnBlur) {
```

Comment:
Hm, I'll mark this for review to investigate.  I can't think of a reason we'd need to set `open` again when it is already open.  Thanks for the note here.